src/axom/sidre/docs/sphinx/data_vs_metadata.rst

```diff
@@ -9,31 +9,33 @@
 Data vs. Metadata Concepts
 ===========================
 
-This section describes examples that illustrate some common Sidre usage 
-patterns involving buffers, groups, and views. In particular, it shows
+This section includes examples that illustrate some common Sidre 
+usage patterns involving buffers, groups, and views. In particular, it shows
 how data associated with a view is related to a buffer, how the lifetimes
 of data associated with a view and the buffer owning the data are related,
 and how the description of data in a view (metadata) is independent of the
 the allocation state of the data in a view and a corresponding buffer. 
 
 .. note:: Object and data creation and destruction is very explicit in Sidre
           to allow maximum flexibility to compose complex operations from 
-          simpler ones. This should become apparent by reading the following 
-          examples. 
+          simpler ones. This should become apparent by following the examples. 
 
-The code snippets shown and described here exist in the file ``axom/src/axom/sidre/examples/sidre_data_vs_metadata.cpp``, which can be built and run to experiment with if you wish.
+The code snippets shown and described here exist in the file 
+``axom/src/axom/sidre/examples/sidre_data_vs_metadata.cpp``, which can be 
+built and run to experiment with if you wish.
 
 The starting point for the first three examples is a simple Sidre datastore 
 group hierarchy in which the root group has two child groups named "A" and "B". 
-This structure is generated as follows:
+This generation of this structure is shown in the following code along with
+some print statements to verify the state is what we expect.
 
 .. literalinclude:: ../../examples/sidre_data_vs_metadata.cpp
    :start-after: _ex_datastore_initial_start
    :end-before: _ex_datastore_initial_end
    :language: C++
 
-At this point, the datastore has zero buffers and the groups have no views
-as we can see by the output of the code::
+As expected, the datastore has zero buffers and the groups have no views,
```

Comment:
```suggestion
As expected, the datastore has no buffers and the groups have no views,
```
"zero" is correct but jarring, especially since it's closely followed by "no views."

Reply:
changed.